pkg/state/types_test.go

```diff
@@ -663,3 +663,49 @@ func TestDeepEqualWithSorting(t *testing.T) {
 		t.Errorf("expected maps to be equal, but they are not")
 	}
 }
+
+func TestConsole(t *testing.T) {
```

Comment:
To make sure it's clear what the test covers.
```suggestion
func TestPluginConsole(t *testing.T) {
```